lib/datastore/transaction.js

```diff
@@ -353,20 +375,25 @@ Transaction.prototype.delete = function(entities) {
  * //-
  * var companyKey = dataset.key(['Company', 123]);
  * var productKey = dataset.key(['Product', 'Computer']);
- * transaction.save([
- *   {
- *     key: companyKey,
- *     data: {
- *       HQ: 'Dallas, TX'
- *     }
- *   },
- *   {
- *     key: productKey,
- *     data: {
- *       vendor: 'Dell'
+ *
+ * dataset.runInTransaction(function(transaction, done) {
```

Comment:
An unfortunate side effect to generating the new JSON schema is that we filter out all private examples, so we'll need to test transactions via `dataset.runInTransaction` now :(((